runners/console/index.ts

```diff
@@ -91,28 +96,32 @@ export class Console extends Runner {
     runInstallCobiGen(step: Step, command: Command): RunResult {
         let result = new RunResult();
         result.returnCode = 0;
-
-        this.executeDevonCommandSync("cobigen", path.join(this.getWorkingDirectory(), "devonfw"), result);
+        if(this.getVariable(this.useDevonCommand)){
+            this.executeDevonCommandSync("cobigen", path.join(this.getWorkingDirectory(), "devonfw"), result);
+        }else{
+            console.warn("Devonfw IDE is not installed");
```

Comment:
I think it would be better to log the warning but execute the command, as well. When the assertion noErrorCode does not throw the error the user may think that the command executed properly and something other caused the error.
The warning is an information for the user that the error is caused by the missing ide.